### ISSUE TITLE
unclear where the canonical docs for lints is

### ISSUE THREAD

**Issue by devoncarew** [dart-lang/linter#3436]
We have the lints documented at both https://dart.dev/tools/linter-rules and https://dart-lang.github.io/linter/. Both locations are mentioned from various places and show up in search results. Are these docs duplicates of each other? Is one more complete? More frequently updated? Is one canonical and the other deprecated?

I think we should de-dup here; either remove one, or, make it clear to users what each is for.

**Comment by parlough** [comment]
The dart.dev version was added so there was an on-site destination (with a relatively similar style to the diagnostics) to link to when lints are referenced across the documentation. 

I think the primary goal should be the standardization of the diagnostics messages. Once that is done, we can add some sort of redirection from the old GitHub pages entries to the diagnostic messages pages.

**Comment by parlough** [comment]
This can be closed now that https://dart.dev/tools/linter-rules is the canonical location for docs and the old GitHub site redirects to there!

We have a bunch of further improvements to the dart.dev hosted version planned, but everything covered by the old site is present on dart.dev now.

**Comment by pq** [comment]
Thanks for pushing on this @parlough!
